CHANGELOG.md

```diff
@@ -192,22 +180,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
-- **CUMULUS-3070**
```

Comment:
Is this intentional? These 3070 changes should not be a part of 13.4.0?

Reply:
Correct. 3070 here in 13.4.0 was intentionally moved to 14.0.0 Changed section.